source/store/AuthContext.js

```diff
@@ -141,6 +143,13 @@ function AuthProvider({ children, initialState }) {
     dispatch(await addProfile());
   }
 
+  /**
+   * Sets any API status message in state.
+   */
+  function handleSetApiStatusMessage(message) {
+    dispatch(setApiStatusMessage(message));
+  }
```

Comment:
Maybe the function could be renamed to reflect the fact in the comment?
`handleSetAnyApiStatusMessageInState` :)